static/adopters/data.js

```diff
@@ -0,0 +1,126 @@
+const projects = [

```

Comment:
Could you double check the project intro on each card? For example, the Helm project intro is not correct
![image](https://github.com/oras-project/oras-www/assets/40452856/11a1650f-d7be-4508-8596-3d3b66530673)